build.fsx

```diff
@@ -242,6 +236,19 @@ Target "BuildRelease" DoNothing
 //--------------------------------------------------------------------------------
 
 //--------------------------------------------------------------------------------
+// Filter out assemblies which can't run on Linux, Mono, .NET Core, etc...
+
+open Fake.EnvironmentHelper
+let filterPlatformSpecificAssemblies (assembly:string) =
```

Comment:
Eh, I'll leave it as-is for now in case someone wants a test that isn't just `.Contains`